web/src/types.ts

```diff
@@ -69,3 +69,22 @@ export type UpdateMessageEvent = {
 // TODO/typescript: Move the User and Stream placeholder
 // types to their appropriate modules.
 export type User = Record<string, never>;
+
+// ADD type Meta and and export to feedback_widget.ts
```

Comment:
Why is this comment so excited about “ADD”?

This file is a temporary home for types belonging to modules that haven’t been converted to TypeScript yet. You’re converting `feedback_widget.ts` to TypeScript, so there’s no reason to put its types here.

Also, both of these types will only ever be referenced from a single declaration. There isn’t any reason to name them from anywhere other than that declaration, so they should just be inlined into the declaration.

Reply:
Actually @PIG208 suggested that we should define all types in `types.ts` and just import them in the modules required for consinstency of the codebase. So this is a bit confusing

Reply:
> One more thing to note here is that we should also export the types that you define here.

I shall clarify. I think I did not state that all types need to be defined in `types.ts`. When you have a direct dependency of a type from another module, you should export the type right in that module.

The comment about the consistency of the code base was just about the usage of `type = xxx` vs. `interface xxx {`.

Reply:
my bad I got it all wrong. Fixing it real soon

Reply:
@PIG208 any thoughts on the changes?

Reply:
@timabbott any final thoughts on this?